aspnetcore/fundamentals/index.md

```diff
@@ -68,25 +69,24 @@ On startup, an ASP.NET Core app builds a *host*. The host encapsulates all of th
 * Dependency injection (DI) services
 * Configuration
 
-There are two different hosts: 
+There are three different hosts:
 
+* .NET Minimal Host
```

Comment:
I'll leave this for Stephen, but I think I've heard this referred to more as the WebApplication host.